tdrs-backend/tdpservice/users/models.py

```diff
@@ -165,38 +174,6 @@ def is_ocio_staff(self) -> bool:
         """Return whether or not the user is in the ACF OCIO Group."""
         return self.is_in_group("ACF OCIO")
 
-    @property
-    def stt(self):
-        """Return the location if the location is an STT."""
-        if self.location and self.location_type.model == "stt":
-            return self.location
-        else:
-            return None
-
-    @stt.setter
-    def stt(self, value):
-        if self.is_regional_staff:
-            raise ValidationError(
```

Comment:
again, this logic was effectively duplicated for the individual actions of `ModelForm` saves, `User.clean` validation, and (here) setting fields directly. Getting rid of the `GenericForeignKey` in favor of individual fields lets us consolidate all the logic with no change to how the model is used.